openmc/region.py

```diff
@@ -637,18 +648,7 @@ def node(self, node):
 
     @property
     def bounding_box(self) -> BoundingBox:
-        # Use De Morgan's laws to distribute the complement operator so that it
-        # only applies to surface half-spaces, thus allowing us to calculate the
-        # bounding box in the usual recursive manner.
-        if isinstance(self.node, Union):
-            temp_region = Intersection(~n for n in self.node)
-        elif isinstance(self.node, Intersection):
-            temp_region = Union(~n for n in self.node)
```

Comment:
I'm a little confused as to how these two cases are different than

```python
temp_region = ~self.node
```

what am I missing?

Reply:
Before, the `Intersection` and `Union` classes did not implement `__invert__`, which meant that you couldn't call `~self.node` if `self.node` was one of those classes. Now that I've implemented `__invert__` on those classes, you can just call `~self.node` (as I've now done for this method).

Reply:
Aye, got my diff turned around. Make perfect sense. Mea culpa.